pkg/tools/cli/loadtest/config/config.go

```diff
@@ -27,8 +29,15 @@ type GlobalConfig = config.GlobalConfig
 // In K8s environment, this configuration is stored in K8s ConfigMap.
 type Data struct {
 	config.GlobalConfig `json:",inline" yaml:",inline"`
+	Addr                string             `json:"addr" yaml:"addr"`
+	Method              string             `json:"method" yaml:"method"`
+	Dataset             string             `json:"dataset" yaml:"dataset"`
+	Concurrency         int                `json:"concurrency" yaml:"concurrency"`
+	ProgressDuration    time.Duration      `json:"progress_duration" yaml:"progress_duration"`
```

Comment:
![badge](https://img.shields.io/badge/review-imo-orange.svg?style=flat-square)
I think it should be string type and parsed in loader_option.go.
also we should add a `GetActualValue` func to read string correctly.